OWNERS_ALIASES

```diff
@@ -21,8 +21,8 @@ aliases:
     - gjtempleton
     - mwielgus
   sig-cli-leads:
-    - eddiezane
     - KnVerey
+    - eddiezane
```

Comment:
Is this change needed ?

Reply:
it's easier to sync the whole OWNERS_ALIASES file from k/k than try to pull in manual updates

Reply:
@ameukam -- This was copied in from k/community's generator, which is sorting ASCIIbetically (so uppercase usernames are sorting in first).